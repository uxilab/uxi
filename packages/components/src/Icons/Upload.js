import React from 'react';
import PropTypes from 'prop-types';
import SvgIcon from '../SvgIcon';
/* eslint-disable max-len */

const Upload = props => (
  <SvgIcon {...props}>
    <svg
      focusable="false"
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 15.75 15.78"
      width="24px"
      height="24px"
    >
      <g id="Layer_2" data-name="Layer 2">
        <g id="Layer_1-2" data-name="Layer 1">
          <polygon points="11.46 3.59 7.88 0 4.29 3.59 5.24 4.54 7.2 2.58 7.2 9.63 8.55 9.63 8.55 2.58 10.51 4.54 11.46 3.59" />
          <path d="M15.75,15.78V9.63h-5l-1.6,1.6H6.55L5,9.63H0v6.15ZM1.35,11h3L6,12.58H9.75l1.6-1.6h3v3.45H1.35Z" />
        </g>
      </g>
    </svg>
  </SvgIcon>
);

Upload.propTypes = {
  color: PropTypes.string,
};

Upload.defaultProps = {
  color: '#6d6d71',
};

export default Upload;
