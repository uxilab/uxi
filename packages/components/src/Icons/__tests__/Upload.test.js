import React from 'react';
import Upload from '../Upload';

describe('Icon : <Upload />', () => {
  it('match snapshot', () => {
    expect(shallow(<Upload />)).toMatchSnapshot();
  });
});

