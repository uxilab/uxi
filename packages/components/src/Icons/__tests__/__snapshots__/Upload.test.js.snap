// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Upload /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 15.75 15.78"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <polygon
          points="11.46 3.59 7.88 0 4.29 3.59 5.24 4.54 7.2 2.58 7.2 9.63 8.55 9.63 8.55 2.58 10.51 4.54 11.46 3.59"
        />
        <path
          d="M15.75,15.78V9.63h-5l-1.6,1.6H6.55L5,9.63H0v6.15ZM1.35,11h3L6,12.58H9.75l1.6-1.6h3v3.45H1.35Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
